src/components/content/order/migrate/useMigrateServiceQuery.ts

```diff
@@ -14,10 +21,42 @@ export function useMigrateServiceQuery() {
     });
 }
 
-export function useMigrateServiceDetailsQuery() {
-    return useMutation({
-        mutationFn: (processInstanceId: string) => {
-            return MigrationService.getMigrationOrderDetailsById(processInstanceId);
+export function useMigrateServiceDetailsPollingQuery(
+    migrationId: string | undefined,
+    isStartPolling: boolean,
+    refetchUntilStates: ServiceMigrationDetails.migrationStatus[]
+) {
+    return useQuery({
+        queryKey: ['getServiceDetailsById', migrationId],
+        queryFn: () => {
+            // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+            return MigrationService.getMigrationOrderDetailsById(migrationId!);
+        },
+        refetchInterval: (query) =>
+            query.state.data && refetchUntilStates.includes(query.state.data.migrationStatus)
+                ? false
+                : deploymentStatusPollingInterval,
+        refetchIntervalInBackground: true,
+        refetchOnWindowFocus: false,
+        enabled: migrationId !== undefined && isStartPolling,
+    });
+}
+
+export function useServiceDetailsPollingQuery(
```

Comment:
@WangLiNaruto we need only one polling here. 
My approach would be -> 
if the migration status is DESTROY_COMPLETED -> then just fetch the service details and display.
if migration status is DESTROY_FAILED -> then show an error but still show the new service details.